packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -1,6 +1,7 @@
 import { declare } from "@babel/helper-plugin-utils";
 import syntaxObjectRestSpread from "@babel/plugin-syntax-object-rest-spread";
 import { types as t } from "@babel/core";
+import { convertFunctionParams } from "@babel/plugin-transform-parameters";
```

Comment:
You need to add this as a dependency in package.json

Reply:
Fixed. For some reason I thought adding it as a dependency wouldn't work because on my previous branch when I tried `npm i -S @babel/plugin-transform-parameters` and `npm i -S @babel/helper-call-delegate` npm threw up due package-lock.json conflict.

Reply:
The correct (by the book) recipe is
```sh
lerna add @babel/helper-call-delegate --scope=@babel/plugin-proposal-object-rest-spread
```
But I feel like it's too verbose and most of time I manually revise `package.json` and then run `yarn lerna bootstrap`.

We should state it clearly in `CONTRIBUTING.md`.